PagePixels Screenshots (Certified Publisher)

Anthony-Dev opened the pull request:
* - Initial Commit


---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

SDPCloud committed:
ManageEngine ServiceDesk Plus Cloud app submission (#2517)

* Partner Center connector - new fields and deal registration actions (#2130)

* Update apiDefinition.swagger.json

* Latest changes

* fix format warning

* X-Caller_Id should be optional

* Update apiDefinition.swagger.json

Review changes -
->Added reg status and sub status fields.
->Update the fields descriptions as per api's details available on Microsoft public site.
->Remove submission related fields.
->Mark registration status field as read only.
->Remove offerId from root.
->Use Enum values in Create deal registration model.
->Create a separate model for update deal registration.
Create a separate action for update deal registration.

* Update apiDefinition.swagger.json

* Update apiDefinition.swagger.json

Add deprecate text in Update a referral by Id action

* Update apiDefinition.swagger.json

Connector status keep as Preview and UpdateReferralById deprecated keep as false but deprecated word in description.

* Update apiDefinition.swagger.json

update deal registration success message.

* Update apiDefinition.swagger.json

added title for registration object

* Bitskout Connector (#2140)

* Bitskout Connector

* Requested changes

* EC-14395 Make it so fields aren't required anymore for updates (#2150)

* Added orderby parameter

* Added orderby parameter

* feat: add x-ms-pageable extension

* feat: add updatenextlink policy instance

* refactor: whitespace refactoring

* Update apiProperties.json

* fix(EC-14395): Integrate the patch schema endpoint

* fix(EC-14395): Add specs for the schema endpoint's responses

* fix(EC-14395): Add required fields to the schema

Co-authored-by: Johannes Heesterman <johannes@elfsquad.io>
Co-authored-by: johannesheesterman <heestermanj@gmail.com>
Co-authored-by: Stan van Rooy <stanvanrooy6@gmail.com>

* ManageEngine ServiceDesk Plus Cloud app submission

* Categories and readme document updated

* Categories updated

---------

Co-authored-by: Farhan <103836444+94farhan@users.noreply.github.com>
Co-authored-by: saferoombiz <32263053+saferoombiz@users.noreply.github.com>
Co-authored-by: Tess Ellenoir Duursma <thomasduursma@outlook.com>
Co-authored-by: Johannes Heesterman <johannes@elfsquad.io>
Co-authored-by: johannesheesterman <heestermanj@gmail.com>
Co-authored-by: Stan van Rooy <stanvanrooy6@gmail.com>
Co-authored-by: Mahbub Murshed <48570820+mamurshe@users.noreply.github.com>
Co-authored-by: SDPCloud <tharanishanth16@gmail.com>

Anthony-Dev committed:
PagePixels Screenshots (Certified Publisher)

* - Initial Commit

Anthony-Dev commented:
@microsoft-github-policy-service agree

certificationteam reviewed (commented):
Hi @Anthony-Dev, 

Can you please refer below Certified connector link and submit only files required. Additionally the logo file is not needed and readme file should include all the required sections. 

https://github.com/microsoft/PowerPlatformConnectors/tree/dev/templates/certified-connectors

Anthony-Dev committed:
Update Readme to match the template's structure, remove the unnecessary logo.png file.

Anthony-Dev commented:
> Hi @Anthony-Dev,
> 
> Can you please refer below Certified connector link and submit only files required. Additionally the logo file is not needed and readme file should include all the required sections.
> 
> https://github.com/microsoft/PowerPlatformConnectors/tree/dev/templates/certified-connectors

Thank you @certificationteam  - I have removed the logo.png file and updated the Readme to match the structure in the provided template.

vmanoharas commented on certified-connectors/PagePixels Screenshots/readme.md:
```diff
@@ -0,0 +1,78 @@
+# PagePixels Screenshots connector
+
```
Please refer the below link and update all required sections. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

vmanoharas commented on certified-connectors/PagePixels Screenshots/apiDefinition.swagger.json:
```diff
@@ -0,0 +1,1857 @@
+          }
+        ],
+        "summary": "Take a real geolocation screenshot of a web page",
+        "description": "Captures a screenshot from a real gelocation. Screenshots are taken through a proxy network of rotating residential IP addresses within the geographical location you specify. Choose from hundreds of countries, cities, and states from around the world.",
```
Please correct the typo "gelocation"

Anthony-Dev replied:
Hi @certificationteam / @vmanoharas - Thanks again.  I've updated the readme to include the Deployment Instructions and fixed the typos in the apiDefinition.

vmanoharas reviewed (commented):
Hi @Anthony-Dev, 

Please address couple of review comments.

Anthony-Dev committed:
Update the readme to include the Deployment Instructions. Update the apiDefinition to fix typos.

vmanoharas commented on certified-connectors/PagePixels Screenshots/apiDefinition.swagger.json:
```diff
@@ -0,0 +1,1857 @@
+{
+  "swagger": "2.0",
```
Please add ""x-ms-connector-metadata":" property, please refer below link and instructions. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json#L25

Please select two categories from the list below  separated by semicolon.
"AI", "Business Management", "Business Intelligence", "Collaboration", "Commerce", "Communication", "Content and Files", "Finance", "Data", "Human Resources", "Internet of Things", "IT Operations", "Lifestyle and Entertainment", "Marketing", "Productivity", "Sales and CRM", "Security", "Social Media", "Website"

Anthony-Dev replied:
Hi @vmanoharas,

Thanks - The x-ms-connector-metadata property has been added and includes the Website, Privacy Policy, and Categories.

vmanoharas reviewed (commented):
Hi @Anthony-Dev, 

Kindly address the review comment.

Anthony-Dev committed:
Add the x-ms-connector-metadata property to the ApiDefinition.swagger.json to define the Website, Privacy Policy, and Categories.

vmanoharas reviewed (approved):
Dear Partner,
 
Congratulations your PR is approved !! We are proud to announce the Brand-New Certification experience for you to certify Power Platform Copilot Connectors & Plugins. 
 
As next steps:
 
Learn about the new certification experience- [Get your Power Platform connector and plugin certified - Overview](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fsubmit-certification&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390456966%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=uWDvs%2BmTbozw1oPc%2FNDNWJ%2FHgnUSfFlRCB1T8UHBh3A%3D&reserved=0)
Package your connector and plugin files- [Prepare Power Platform connector and plugin files for certification](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-submission&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390465418%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=pDuv4bnya1eP3Rrs3p4ArNOXw%2FQipWU87q5oZ0VSIe0%3D&reserved=0)
Validate the package for structure before submitting the package for certification to Partner Center - [Run Package Validator tool](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-submission%23step-7-validate-the-package-for-structure&data=05%7C02%7Cshefaali.patankar%40microsoft.com%7C6f0c4adf688f42a7107908dcb6c72a8c%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638586213737574171%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=9QGRIRe%2FZ%2B1sgUF5JQjY2DeuyxDt8f8lDwpGFEaaBmg%3D&reserved=0)
Initiate a certification request in Partner Center- [Verified publisher certification process](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fsubmit-for-certification&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390471799%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=zbm2px5VZ2ABLBFcORH6tu1fsC9KWdXkP1XJFyluMzw%3D&reserved=0)
Ensure your connector & plugin files comply to the Marketplace policies here- [5000 Power Platform Connector Policies for Marketplace](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Flegal%2Fmarketplace%2Fcertification-policies%235000-power-platform-connectors&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390477736%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=h0pI5J5buo82xxvnBVmrAN8K%2F%2BK8xZlUdcSwzWoqWq0%3D&reserved=0) and also [1000 Marketplace policies](https://learn.microsoft.com/en-us/legal/marketplace/certification-policies#100-general)(as applicable)
Test your connector post certification to provide go-live for deployment- [Testing Guidelines](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-testing&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390483743%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=Z8pSnhCArFRzEr373o29qM5h5VbV1nlN2gVbv%2Blk1aE%3D&reserved=0)
If you wish to Update your connector or plugin in future- [Updating Guidelines](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-updates&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390489681%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=CvxTWJFE3bTFeJ3nDJoCSpzvimhVV9go4cBG%2FQ4gMvo%3D&reserved=0)
 
For any queries or concerns please get in touch with us on [certificationteam@microsoft.com](mailto:certificationteam@microsoft.com)
 
Many thanks,
Power Platform
Copilot connector & plugin certification team

vmanoharas reviewed (dismissed):
Dear Partner,

Congratulations your PR is approved !! We are proud to announce the Brand-New Certification experience for you to certify Power Platform Copilot Connectors & Plugins.

As next steps:

Learn about the new certification experience- [Get your Power Platform connector and plugin certified - Overview](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fsubmit-certification&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390456966%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=uWDvs%2BmTbozw1oPc%2FNDNWJ%2FHgnUSfFlRCB1T8UHBh3A%3D&reserved=0)
Package your connector and plugin files- [Prepare Power Platform connector and plugin files for certification](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-submission&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390465418%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=pDuv4bnya1eP3Rrs3p4ArNOXw%2FQipWU87q5oZ0VSIe0%3D&reserved=0)
Validate the package for structure before submitting the package for certification to Partner Center - [Run Package Validator tool](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-submission%23step-7-validate-the-package-for-structure&data=05%7C02%7Cshefaali.patankar%40microsoft.com%7C6f0c4adf688f42a7107908dcb6c72a8c%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638586213737574171%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=9QGRIRe%2FZ%2B1sgUF5JQjY2DeuyxDt8f8lDwpGFEaaBmg%3D&reserved=0)
Initiate a certification request in Partner Center- [Verified publisher certification process](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fsubmit-for-certification&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390471799%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=zbm2px5VZ2ABLBFcORH6tu1fsC9KWdXkP1XJFyluMzw%3D&reserved=0)
Ensure your connector & plugin files comply to the Marketplace policies here- [5000 Power Platform Connector Policies for Marketplace](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Flegal%2Fmarketplace%2Fcertification-policies%235000-power-platform-connectors&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390477736%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=h0pI5J5buo82xxvnBVmrAN8K%2F%2BK8xZlUdcSwzWoqWq0%3D&reserved=0) and also [1000 Marketplace policies](https://learn.microsoft.com/en-us/legal/marketplace/certification-policies#100-general)(as applicable)
Test your connector post certification to provide go-live for deployment- [Testing Guidelines](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-testing&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390483743%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=Z8pSnhCArFRzEr373o29qM5h5VbV1nlN2gVbv%2Blk1aE%3D&reserved=0)
If you wish to Update your connector or plugin in future- [Updating Guidelines](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-updates&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390489681%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=CvxTWJFE3bTFeJ3nDJoCSpzvimhVV9go4cBG%2FQ4gMvo%3D&reserved=0)

For any queries or concerns please get in touch with us on [certificationteam@microsoft.com](mailto:certificationteam@microsoft.com)

Many thanks,
Power Platform
Copilot connector & plugin certification team

vmanoharas commented:
Hi @Anthony-Dev, 

Please change the branch from master to dev, as we cannot merge this code. 

You have used master branch

<img width="218" height="80" alt="image" src="https://github.com/user-attachments/assets/2212489a-090a-4850-931b-5047e6ff437d" />

Please use dev branch. 

<img width="187" height="74" alt="image" src="https://github.com/user-attachments/assets/d2843272-f1e3-475a-abb4-6200898e7a82" />

Anthony-Dev commented:
> Hi @Anthony-Dev,
> 
> Please change the branch from master to dev, as we cannot merge this code.
> 
> You have used master branch
> <img alt="image" width="218" height="80" src="https://private-user-images.githubusercontent.com/137378994/481001570-2212489a-090a-4850-931b-5047e6ff437d.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU5MDE3NTksIm5iZiI6MTc1NTkwMTQ1OSwicGF0aCI6Ii8xMzczNzg5OTQvNDgxMDAxNTcwLTIyMTI0ODlhLTA5MGEtNDg1MC05MzFiLTUwNDdlNmZmNDM3ZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyMlQyMjI0MTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05ZTI0OTU0ZjM0ODJhMzAyZWRlMWViMDA3NGNkN2RkOGRhZDNhMTU3YjA5OTdkZmY5YzBhYjAzMmM3ZjEyZWUxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.xwvXXE_9hNMtupS_lZAMmVRjYhn0iCdxayJ4AcFzAC4">
> 
> Please use dev branch.
> <img alt="image" width="187" height="74" src="https://private-user-images.githubusercontent.com/137378994/481001894-d2843272-f1e3-475a-abb4-6200898e7a82.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU5MDE3NTksIm5iZiI6MTc1NTkwMTQ1OSwicGF0aCI6Ii8xMzczNzg5OTQvNDgxMDAxODk0LWQyODQzMjcyLWYxZTMtNDc1YS1hYmI0LTYyMDA4OThlN2E4Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyMlQyMjI0MTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNGM0NTVkOGNhZDU4M2VlOTgwMWE3Njg3NjQyZTM2MWRkYmVlYWVjOWQwNjc1OWQ0OGY2ZWQ5ZDg3ZmM2NzhmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.GJT0PHzuIW1_a0qbfJ5TSvts2LKmL14AadA0nBEAsmg">

Hi @vmanoharas - Thanks, I’ve updated the PR to target the microsoft:dev branch instead of master. I can also recreate this PR from the fork's `dev` branch if it would make things easier, please let me know.

Anthony-Dev commented:
Hey @vmanoharas / @certificationteam - I went ahead and created another PR request from PagePixels:dev into microsoft:dev explicitly, in case it is helpful.  

https://github.com/microsoft/PowerPlatformConnectors/pull/3975

vmanoharas commented:
> Hey @vmanoharas / @certificationteam - I went ahead and created another PR request from PagePixels:dev into microsoft:dev explicitly, in case it is helpful.
> 
> #3975

Thank you @Anthony-Dev for creating new PR.  Kindly close this PR.
